matrix_client/room.py

```diff
@@ -222,6 +222,50 @@ def send_audio(self, url, name, **audioinfo):
         return self.client.api.send_content(self.room_id, url, name, "m.audio",
                                             extra_information=audioinfo)
 
+    def send_call_invite(self, call_id, sdp, version, lifetime):
```

Comment:
These don't offer any additional abstraction over methods defined in api class. Until we have a reason to put methods here that make interacting with the api methods easier, probably just delete these methods.